meta-lmp-base/recipes-support/htpdate/htpdate/default.conf

```diff
@@ -0,0 +1 @@
+HTPDATE_ARGS="-st www.example.com"
```

Comment:
Since this is a time update, shouldn't this be a real destination?

Reply:
www.example.com is a valid destination, which is ok to be used as default, customers can change to any other address that might be valid for their own network.